app/models/extra_question.rb

```diff
@@ -0,0 +1,39 @@
+class ExtraQuestion < ApplicationRecord
+  validates_presence_of :question, :data_type
+
```

Comment:
What is the association?

Reply:
I am not sure rails associations would work since the ID is stored as part of a serialized hash in questionnaire